docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -148,11 +192,8 @@ kind: AzureMachineTemplate
 metadata:
   name: node-os-image-caching-machine-template
   namespace: default
-spec:
-  template:
-    spec:
-      nodeCachingMode:
-        interval: 24h
+  annotations:
+    nodeCachingInterval: 24h
```

Comment:
Annotation keys tend to be formatted a bit differently than field names under `spec` (https://kubernetes.io/docs/reference/labels-annotations-taints/), e.g. https://github.com/nojnhuh/cluster-api-provider-azure/blob/2bc1b22e429ee160533a4379a6b2f87cc698b622/azure/const.go

Reply:
Ended up moving this around to the point where I think this isn't as relevant currently.